invenio_app_ils/ui/src/invenio_app_ils/common/api/items/__tests__/serializer.js

```diff
@@ -11,19 +11,16 @@ describe('Patron loans serialization tests', () => {
       created: stringDate,
       metadata: {
         item_pid: '123',
```

Comment:
The description of the tests I think is outdated. I understand that metadata are not empty and also after the serialization we keep also the `created` other than `id` and `updated`.